### DISCUSSION TITLE
New Building Idea: Cultivator

### DISCUSSION THREAD

**Discussion by f1vetails** [ldtteam/minecolonies#8769] [Ideas]
So I've been mulling this over for a long time. 
So the building would be the Cultivator: This building will make Kelp, Seagrass, and Corals (in a similar fashion to the flower shop.) It is a coastal building, that has Composted dirt 12x per level (like the plantation.) 4 each for Corals kelp and sea grass, and uses compost to make/plant these items. 

Similarly it would be locked behind research, like the plantation.
It should also have compost barrels and a furnace in its build (to make dried Kelp as a craft, and bundle it)

The research would be 64 Kelp.

I've enclosed an image of the level 5 building built in my style (High Magic):
![2022-11-13_12 41 41](https://user-images.githubusercontent.com/118127957/201536296-1675f1f3-2b12-4b78-b612-65854d5023f3.png)

**Comment by MotionlessTrain** [comment]
Please make an issue at https://github.com/ldtteam/minecolonies-features/issues for minecolonies suggestions

**Reply by f1vetails** [reply]
Oh. It didn't seem like the right way 'issue' threw me off.
